python/src/worker/helpers/send_status_updates.py

```diff
@@ -0,0 +1,52 @@
+from worker_status_codes import (
+    DOWNLOAD_EXPERIMENT,
+    LOAD_EXPERIMENT,
+    STARTED_TASK,
+    COMPRESSING_TASK_DATA,
+    UPLOADING_TASK_DATA,
+    FINISHED_TASK,
+)
+
+
+# format_task_name is a helper function that takes a request and returns a formatted task name
+# if the request has a name, otherwise it returns an empty string
+def format_task_name(request):
+    try:
+        task_name = request["body"]["name"]
+    except KeyError:
+        return ""
+    # Remove "Get" from the name
+    name_without_get = task_name.replace("Get", "")
+
+    # Insert a space before capital letters, then strip leading/trailing spaces
+    final_name = " ".join(
+        [c if not c.isupper() else f" {c}" for c in name_without_get]
+    ).strip()
+
+    return final_name
+
+
+def format_user_message(status_code, request):
+    task_name = format_task_name(request)
+    if status_code == DOWNLOAD_EXPERIMENT or status_code == LOAD_EXPERIMENT:
+        return "Accessing the Seurat object for your analysis"
```

Comment:
You can also return those codes from here to the UI instead of having them there too. That way when you receive an error in the UI, you will understand where it comes from if its something like `WORKER_DOWNLOAD_EXPERIMENT`.

Reply:
yes this is done already, this function only formats the name. See this below:
```
            "type": "WorkResponse",
            "status_code": status_code,
            "user_message": format_user_message(status_code, request),
```